google/cloud/internal/backoff_policy.h

```diff
@@ -103,19 +108,18 @@ class ExponentialBackoffPolicy : public BackoffPolicy {
    *     `std::chrono::duration<>` (in brief, the underlying arithmetic type
    *     used to store the number of ticks), for our purposes it is simply a
    *     formal parameter.
-   * @tparam Rep2 similar formal parameter for the type of @p maximum_delay.
    * @tparam Period1 a placeholder to match the Period tparam for
    *     @p initial_delay's type, the semantics of this template parameter are
    *     documented in `std::chrono::duration<>` (in brief, the length of the
    *     tick in seconds, expressed as a `std::ratio<>`), for our purposes it
    *     is simply a formal parameter.
    * @tparam Period2 similar formal parameter for the type of @p maximum_delay.
+   * @tparam Rep2 similar formal parameter for the type of @p maximum_delay.
```

Comment:
These are still out of order with the params below (switch doc placement for Period2 and Rep2).

Reply:
Fixed.